### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by IrynaBarna**
Please  review @rbbratta @seungweonpark @NazarTkachuk @lijenni @orestkreminskyi @esmacgil 

Signed-off-by: Iryna Barna <irynax.barna@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,57 @@
...
+### Design considerations
 
-## Hardware-related libraries:
-* dev_* - modules which contain classes to handle particular environment type;
-* powerboard - startup/shutdown of the devices using SNMP via APC power boards;
+**System Requirements and Dependencies**
+
+The framework is designed for use in Linux systems which have Python programming tools and the necessary libraries installed. The framework has been tested on following systems:
+
+* Ubuntu 12.04 LTS running Python 3.4.0, py.test 2.9.2 and higher versions.
```
Comment (rbbratta) [inline README.md]: we don't really want to support Ubuntu 12.04 so please remove this reference so people don't expect 12.04 support.

**Comment by rbbratta** [comment]
Review scrub: please address comment

File: README.md (hunk 1 above)
Comment (seungweonpark) [reply README.md]: @IrynaBarna Why don't we remove the line#15?

Comment (IBarna) [inline README.md]: I will remove mentions about Ubuntu 12.04 in README and html docs in next commit.

Comment (IrynaBarna) [reply README.md]: I will remove mentions about Ubuntu 12.04 in README.md file and TAF html in next commit.

**Comment by rbbratta** [comment]
please squash the commits and remove the commit message about Ubuntu 12.04

File: README.md [hunk 2]
```diff
@@ -25,6 +56,166 @@ TAF package consists of two folders: ./taf-repo and ./testcases-repo. taf-repo f
...
+
+    `docker ps`
+- See all containers and their status:
+
+    `docker ps -a`
+
+
+### Simple test execution
+
+> (TODO: Create pull request in test repo with simple localhost config and linux_host test case)
```
Comment (lijenni) [inline README.md]: Does this still need to be done?

Comment (IrynaBarna) [reply README.md]: Development  of this test scenario is in progress.

Comment (seungweonpark) [reply README.md]: @lijenni Observed that you correct the project name, so we'd better clarify the name of the project. How about to use TAF as project name, and taf3 as a version which supports Python 3?

**Review by lijenni — APPROVED** [review]

**Pull request merged**